tests/107.solrresponse_getrawresponseheaders.phpt

```diff
@@ -36,7 +36,7 @@ $query_response = $client->query($query);
 $headers = $query_response->getRawResponseHeaders();
 print_r($headers);
 ?>
---EXPECT--
+--EXPECTF--
 HTTP/1.1 200 OK
 Content-Type: application/xml; charset=UTF-8
-Transfer-Encoding: chunked
\ No newline at end of file
+Content-Length: %s
```

Comment:
I am not sure about this one. This header change is the result of going from Solr-5 to Solr-6. 

In Solr-9 we have MANY more headers.

Should we perhaps just extract the keys out from `$headers` and filter them to check only for the common headers like `HTTP.* 200 OK` and `Content-Type`?

I think the purpose of this test is not to test the exact header values but just making sure we actually get some headers from the response.

Reply:
Good idea, I have changed the test to filter these two headers.